src/extras/CoolBoardSensors.cpp

```diff
@@ -155,7 +155,7 @@ bool CoolBoardSensors::config() {
   config.set<bool>(json, "vbat", this->vbatActive);
   config.set<bool>(json, "vbat", this->vbatActive);
   config.set<bool>(json, "soilMoisture", this->soilMoistureActive);
-  config.set<bool>(json, "wallMoisture", this->soilMoistureActive);
+  config.set<bool>(json, "wallMoisture", this->wallMoistureActive);
```

Comment:
**WHY is this _here_ ?** This has **nothing** to do with the current changes !!!